dvr_scan/scanner.py

```diff
@@ -791,11 +800,24 @@ def scan(self) -> Optional[DetectionResult]:
                 )
             result = detector.update(frame.frame_bgr)
             frame_score = result.score
+
+            box_width = cv2.boundingRect(result.subtracted)[2] * self._downscale_factor
```

Comment:
Calculating a bounding rectangle does have some performance cost associated with it.  Can we make the new parameters either optional, or if the max is >= 1.0, avoid calculating the boundingRect?

Reply:
Done!